### PR TITLE
Bump to logr v1.2.0

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	log := glogr.New()
+	log.WithName("hello").WithName("world").Info("thanks for the fish")
+}
+
+func ExampleNewWithOptions() {
+	initGlog()
+	log := glogr.NewWithOptions(glogr.Options{LogCaller: glogr.Error})
+	log.Info("Info log with LogCaller=Error")
+	log.Error(nil, "Error log with LogCaller=All")
+}
```
Comment (pohly) [inline example_test.go:68]: Is there a way to see the output?

After a brief search I came up with none.

**Comment by thockin** [comment]
Not that I can see - the timestamp is not disableable or fakeable.

File: example_test.go (hunk 1 above)
Comment (thockin) [reply example_test.go:68]: Not that I can see - the timestamp is not disableable or fakeable.

Comment (thockin) [reply example_test.go:68]: I could add it as a comment, but not a standard "Output" comment (ie. not test-verifiable)

Comment (pohly) [reply example_test.go:68]: I know we can't test it, but I was hoping that `go test` at least could show the output on demand. But apparently there is no such feature.

Comment (pohly) [reply example_test.go:68]: `go run ./example` is a bit better in that regard.

**Comment by thockin** [comment]
Updated

**Review by pohly — APPROVED** [review]
Let's not forget to mention the off-by-one bug fix in the release notes...

**Pull request merged**